engine/runtime/src/main/java/org/enso/interpreter/node/expression/builtin/Polyglot.java

```diff
@@ -3,4 +3,8 @@
 import org.enso.interpreter.dsl.BuiltinType;
 
 @BuiltinType
-public class Polyglot extends Builtin {}
+public final class Polyglot extends Builtin {
+  public Polyglot() {
```

Comment:
I'd say we have way too many `Builtin`s. Not sure why `Polyglot` or `Debug` should be builtin types at all!?